index.js

```diff
@@ -58,21 +60,19 @@ module.exports = function (params) {
       pattern: new RegExp('^' + entry.outPath.replace(/[-/\\^$&*+?.()|[\]{}]/g, '\\$&')),
     }
   });
-
   return {
     addMultiEntry: function (config) {
       if(!entries || !entries.length){
         return config;
       }
-      // Mulitple Entry JS
+      // Multiple Entry JS
       const defaultEntryHTMLPlugin = config.plugins.filter(function(plugin){
         return plugin.constructor.name === 'HtmlWebpackPlugin'
       })[0];
       defaultEntryHTMLPlugin.options.chunks = [defaultEntryName];
       const necessaryEntry = config.entry.filter(function(file){
-        return file !== appIndexJs;
+        return !appIndexes.includes(file);
```

Comment:
Array.includes Supported from version v6.4: https://node.green/#ES2015-functions-arrow-functions

CreateReactApp Support from v6.x: https://github.com/facebook/create-react-app/tree/v1.0.0

Since we not using babel transform source code, please using plain javascript "Array.indexOf" instead, Thanks a lot!

Reply:
According to that site Array.includes is supported from 6.0.0 onwards (hover over the 6.4.0 version label)